Sources/MapboxNavigation/DayStyle.swift

```diff
@@ -56,6 +56,13 @@ extension UIFont {
     // General styling
     fileprivate class var defaultPrimaryText: UIFont { get { return UIFont.systemFont(ofSize: 26) } }
     fileprivate class var defaultSecondaryText: UIFont { get { return UIFont.systemFont(ofSize: 16) } }
+
+    /**
+     List of Mapbox Maps font names to be used for any symbol layers added by the Navigation SDK.
+     These are used for features such as Route Duration Annotations that are optionally added during route preview.
+     See https://docs.mapbox.com/ios/maps/api/6.3.0/customizing-fonts.html for more information about server-side fonts.
```

Comment:
Where will the (not terribly intuitive) font fallback behavior be documented for map SDK v10? Will it be in the v10 API reference or a guide on docs.mapbox.com?

/cc @mapbox/docs

Reply:
That is a very good question. I looked for the font documentation in the v10 beta documentation and it isn't present. I went with the less desired approach of mentioning the v6.3 documentation until the place in the v10 documentation is sorted out.

Perhaps I should write a ticket to update this comment when the docs for Maps v10 are finalized.